packages/react/src/components/notebook/NotebookCommands.ts

```diff
@@ -274,13 +275,38 @@ export const NotebookCommands = (
     execute: () => NotebookActions.splitCell(notebookPanel.content),
   });
   commandRegistry.addCommand(cmdIds.undo, {
```

Comment:
not sure whats going on with the indentation here

Reply:
probably related to the change below in this file. I you don't know, that is fine, I can merge as such and look after.

Reply:
im using a linter with prettier, but its looking fine on my end but showing up with weird indentation on the pr.

Reply:
prolly a github display issue. will resolve this.